### PR TITLE
adds specific icons for various event sources

### PR DESCRIPTION
**Pull request by invincibleJai**
- adds specific icons for various event sources

Tracks:
- https://jira.coreos.com/browse/ODC-2156

Screenshot 2019-11-04 at 11 22 56 PM

### CODE REVIEW

**Comment by christianvogt** [comment]
@invincibleJai I'm wondering if the catalog-item-icons is the correct place for these icons or not. They aren't being used by the catalog. Maybe we're better off just placing them in the knative plugin itself.

cc @spadgett thoughts?

**Comment by joshuawilson** [comment]
I think that since knative/serving is an operator and not inherently a part of OpenShift then those icons should go in the knative-plugin.

**Comment by invincibleJai** [comment]
@christianvogt @joshuawilson have updated it, PTAL

File: frontend/packages/dev-console/src/components/topology2/components/nodes/EventSource.tsx [hunk 1]
```diff
@@ -34,6 +35,22 @@ const EventSource: React.FC<EventSourceProps> = ({
   const { width, height } = element.getBounds();
   const size = Math.min(width, height);
   const { data } = element.getData();
+  const getEventSourceIcon = () => {
```
Comment (sahil143) [inline frontend/packages/dev-console/src/components/topology2/components/nodes/EventSource.tsx]: Can we create this as a separate utility?

**Comment by invincibleJai** [comment]
@christianvogt @sahil143 have made the changes, PTAL

**Pull request merged**